e2e/chart-snapshots.spec.ts

```diff
@@ -0,0 +1,40 @@
+import intConfigs from "../app/test/__fixtures/config/int/configs";
+
+import { test, sleep } from "./common";
+
+// Right now the CI app server runs connected to int.lindas.admin.ch
+const configs = intConfigs.map((x) => ({ env: "int", ...x }));
+
+const viewports = {
+  "ipad-mini, portrait": {
+    width: 768,
+    height: 1024,
+  },
+  "iphone-8, portrait": {
+    width: 375,
+    height: 667,
+  },
+};
+
+test("Chart Snapshots", async ({ page }) => {
+  test.setTimeout(5 * 60 * 1000);
```

Comment:
Because the Heavy metal dataset is very long to load, and all the charts were tested in the same test, since Cypress was slow to start new tests. 

Now each chart is is its own test, which is better for reporting failures, and lets us adjust to timeout to 1m (which is very long, but we have to do that until we figure out the performance problems of the Heavy metal dataset).